### ISSUE TITLE
same for less-lint?

### ISSUE THREAD

**Issue by AngeIII** [idok/scss-lint-plugin#14]
Is there available same realization for LESS files?

**Comment by idok** [comment]
Sorry, I'm not familiar with a linter for LESS files.

**Comment by bobrocke** [comment]
As I understand it (probably poorly) LESS will lint its files with the right option set.

**Comment by AngeIII** [comment]
@bobrocke yes, it actually check for parsing errors. but nothing more options like variables on top, sorting for properties, indentations and etc.

**Comment by idok** [comment]
I've looked into it and it is disappointing

**Comment by bobrocke** [comment]
Well, some linting is better than no linting. Right?
